.github/workflows/ci.yml

```
@@ -183,13 +183,23 @@ jobs:
           docker compose up -d relay-1 relay-2 --no-build
           docker compose up -d gateway --no-build
           docker compose up -d client --no-build
+      - name: Add 10ms simulated latency
+        run: |
+          # Add 10ms latency to the client
+          docker compose exec -d client tc qdisc add dev eth0 root netem delay 10ms
+          # Add 10ms latency to the gateway
+          docker compose exec -d gateway tc qdisc add dev eth0 root netem delay 10ms
+          # Add 10ms latency to the relay-1
+          docker compose exec -d relay-1 tc qdisc add dev eth0 root netem delay 10ms
+          # Add 10ms latency to the relay-2
+          docker compose exec -d relay-2 tc qdisc add dev eth0 root netem delay 10ms
```

```suggestion
          docker compose exec -d client tc qdisc add dev eth0 root netem delay 10ms
          docker compose exec -d gateway tc qdisc add dev eth0 root netem delay 10ms
          docker compose exec -d relay-1 tc qdisc add dev eth0 root netem delay 10ms
          docker compose exec -d relay-2 tc qdisc add dev eth0 root netem delay 10ms
```

I'd find this easier to read, especially when there is no syntax highlighting. I think the step name is enough documentation here.